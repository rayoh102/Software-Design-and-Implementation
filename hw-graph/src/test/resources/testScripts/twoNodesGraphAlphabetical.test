# Same as twoNodeGraph.test, but reverse the order of node adding to test if it prints out in alphabetical order

# Create a graph
CreateGraph twoNodesGraphAlphabetical

# Add two nodes in reverse alphabetical order
AddNode twoNodesGraphAlphabetical b
AddNode twoNodesGraphAlphabetical a

# Print the nodes in the graph, the outgoing edges from a and the outgoing edges from b
ListNodes twoNodesGraphAlphabetical
ListChildren twoNodesGraphAlphabetical a
ListChildren twoNodesGraphAlphabetical b