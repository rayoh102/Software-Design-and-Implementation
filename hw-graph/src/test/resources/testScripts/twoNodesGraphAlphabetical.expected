# Same as twoNodeGraph.test, but reverse the order of node adding to test if it prints out in alphabetical order

# Create a graph
created graph twoNodesGraphAlphabetical

# Add two nodes in reverse alphabetical order
added node b to twoNodesGraphAlphabetical
added node a to twoNodesGraphAlphabetical

# Print the nodes in the graph, the outgoing edges from a and the outgoing edges from b
twoNodesGraphAlphabetical contains: a b
the children of a in twoNodesGraphAlphabetical are:
the children of b in twoNodesGraphAlphabetical are: